src/components/RecentlyFlights/RecentlyFlights.js

```diff
@@ -0,0 +1,227 @@
+import { useState, useEffect, useRef } from 'react';
+
+import { ImLoop } from 'react-icons/im';
+import { FiChevronLeft } from 'react-icons/fi';
+import { FiChevronRight } from 'react-icons/fi';
+
+import styled from 'styled-components';
+
+const TOTAL_SLIDES = 1;
+
+const RecentlyFlights = () => {
+  const [currentSlide, setCurrentSlide] = useState(0);
+  const slideRef = useRef(null);
+
+  const flightTickets = [
+    {
+      id: 1,
+      trip_type: '왕복',
+      departure: '김포 (GMP)',
+      arrival: '부산 (PUS)',
+      date_departure: '1월 12일',
+      date_arrival: '1월 20일',
+      personnel: '성인',
+      personnel_count: 2,
+      seat_type: '전체',
+    },
+    {
+      id: 2,
+      trip_type: '왕복',
+      departure: '김포 (GMP)',
+      arrival: '부산 (PUS)',
+      date_departure: '1월 12일',
+      date_arrival: '1월 20일',
+      personnel: '성인',
+      personnel_count: 2,
+      seat_type: '전체',
+    },
+  ];
+
+  const nextBtn = () => {
+    if (currentSlide === 0) {
+      setCurrentSlide(1);
+    } else {
+      setCurrentSlide(TOTAL_SLIDES - 1);
+    }
+  };
+
+  const prevBtn = () => {
+    if (currentSlide === 0) {
+      setCurrentSlide(1);
+    } else {
+      setCurrentSlide(TOTAL_SLIDES - 1);
+    }
+  };
+
+  // const hideArrows = e => {
+  //   if (flightTickets.length <= 3) {
+  //     // 화살표 다 숨김
+  //     e.style.display = 'none';
+  //   }
+  // };
+
+  useEffect(() => {
+    slideRef.current.style.transition = 'all 0.5s ease-in';
+    slideRef.current.style.transform = `translateX(-${currentSlide}00%)`;
+  }, [currentSlide]);
+
+  // max 6개인 캐러셀, 컨텐츠가 하나 더 추가된다면 배열의 첫번째 요소를 제거
+
+  return (
+    <Container>
+      <Title>최근 검색한 항공권</Title>
+      <CarouselFlightsWrap>
+        <ul ref={slideRef}>
+          {flightTickets.map(card => {
+            return (
+              <TicketItem key={card.id}>
+                <FlightContent>
+                  <label>{card.trip_type}</label>
+                  <span className="flightRoute">
```

Comment:
```suggestion
                  <FlightRoute>
```
어차피 이름지을거라면 이렇게 하면 어떨까요?

Reply:
style 주기 위해 어차피 className 붙일거라면 컴포넌트로 만들라는 의미시군요 !!
분명 종택님이 하나의 컴포넌트, 하나의 역할 및 책임이라고 하셨는데도 불구하고 ... 😇 소심하게 어떤 부분은 네스팅으로 묶어버렸습니다.. 
 반성합니다... 후다닥 수정하겠습니다.